website/public/.htaccess

```diff
@@ -1,8 +1,20 @@
 # Copyright (C) 2020  Matthew "strager" Glazar
 # See end of file for extended copyright information.
 
-# Optimize loads of the main page and press release pages using HTTP/2 push.
+AddType application/wasm .wasm
+AddType text/javascript .mjs
+
+Header always set Strict-Transport-Security "max-age=63072000; includeSubDomains"
+Header always set X-Frame-Options "SAMEORIGIN"
+Header always set Referrer-Policy "same-origin"
+Header always set X-Content-Type-Options "nosniff"
+Header always add Content-Security-Policy "default-src 'self'; script-src 'self' 'unsafe-inline' 'unsafe-eval'; style-src 'self' 'unsafe-inline'; media-src 'self' ssl.gstatic.com"
```

Comment:
Must fix: gstatic? Why is gstatic here? We shouldn't be loading anything from gstatic!

Reply:
I am not sure but one of the page was loading mp4 file from gstatic

Reply:
I will remove gstatic

Reply:
I am sorry it was one of the google chrome extensions I am using.